src/main/java/com/internet/shop/controller/DeleteUserController.java

```diff
@@ -0,0 +1,22 @@
+package com.internet.shop.controller;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteUserController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        long id = Long.parseLong(req.getParameter("id"));
+        userService.deleteById(id);
```

Comment:
if we add shoppingCart by default in RegistrationController it makes sense to delete it here when we delete user

Reply:
done